cli/config.go

```diff
@@ -24,10 +24,12 @@ func Config(ctx context.Context, logger *zap.Logger, cfg *config.Config, service
 		Short:   "manage keploy configuration file",
 		Example: "keploy config --generate --path /path/to/localdir",
 		PreRunE: func(cmd *cobra.Command, _ []string) error {
-			return cmdConfigurator.ValidateFlags(ctx, cmd)
+			if err := cmdConfigurator.ValidateFlags(ctx, cmd); err != nil {
```

Comment:
since we are not doing anything here, it can be left unchanged to `return cmdConfigurator.ValidateFlags(ctx, cmd)` as in the previous commit.